addon/components/listbox/-button.hbs

```diff
@@ -0,0 +1,18 @@
+{{#let (element (or @as 'button')) as |Tag|}}
+  <Tag
+    role={{if @role @role 'button'}}
+    id='{{@guid}}-button'
+    aria-haspopup='listbox'
+    aria-controls={{@guid}}
+    aria-labelledby='{{@guid}}-label {{@guid}}-button'
+    {{on 'click' @handleButtonClick}}
+    {{on 'keydown' @handleKeyDown}}
+    {{on 'keypress' @handleKeyPress}}
+    {{aria "expanded" value=@isOpen removeWhen=@disabled}}
```

Comment:
I don't think this `{{aria}}` helper is necessary; if the attribute is set to a false-y value, it will be removed from the DOM node. We should be able to create an appropriate value that we can set `aria-expanded` to to avoid needing this custom behavior

Reply:
It seems that those three states should be, [according to MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaExpanded):

1. The string `"true"`
2. The string `"false"`
3. The string `"undefined"`

Maybe we don't need to remove the attribute completely at all?

If we _do_ want to remove attribute entirely in the third case, we can set it to the value `undefined` (rather than `"undefined"`) and the attribute will be removed 

> Don't you think we should follow the same pattern ? I assumed that guys from react attach importance to accessibility.

Absolutely! We just don't need to overcomplicate things to achieve this 😬

Reply:
https://github.com/GavinJoyce/ember-headlessui/pull/76/commits/11c9843bce96d08007f499c4e3a66574115e1d6f removes the `aria` modifier.